app/models/defendant.rb

```diff
@@ -30,7 +30,7 @@ def arrest_summons_number
   end
 
   def offences
-    body['offences'].map { |offence| Offence.new(body: offence) }
+    body['offences']&.map { |offence| Offence.new(body: offence) }
```

Comment:
Similar to `defendants` shouldn't we assume that we have this info anyway?

Reply:
Removed